actix/src/handler.rs

```diff
@@ -602,3 +528,17 @@ SIMPLE_RESULT!(f32);
 SIMPLE_RESULT!(f64);
 SIMPLE_RESULT!(String);
 SIMPLE_RESULT!(bool);
+
+// Helper trait for send one shot message from Option<Sender> type.
+// None and error are ignored.
+trait OneshotSend<M> {
```

Comment:
This trait seems a bit unnecessary.

Reply:
It's private and reduce loc. There are a lot duplicate if let in handler module